components/bl-draggable-list-component/src/helpers/id.js

```diff
@@ -0,0 +1,10 @@
+export function enrichWithUids(list) {
+  return list.map((item, i) => ({ ...item, uid: i }));
+}
+
+export function omitUids(list) {
+  return list.map(item => {
+    delete item.uid;
```

Comment:
this mutates an `item`, is this by design?

Reply:
To work with drag and drop, we need some unique identifier, when we receive the list, we add uid to all elements, and we return it to the logic without uid , that is, in the format in which we received the data.

Reply:
I understand that, but my questions about the mutation of each `item`

in case with `enrichWithUids ` function you create a new objects but here you mutate

Reply:
export function omitUids(list) {
  return list.map(item => ({ label: item.label, value: item.value }));
}

That's right?